apps/dashboard/app/helpers/batch_connect/sessions_helper.rb

```diff
@@ -195,17 +195,37 @@ def status_context(session)
     end
   end
 
+
+  def terminate_delete(session)
+    if Configuration.terminate_session_enabled && !session.completed?
+      terminate(session)
+    else
+      delete(session)
+    end
+  end
```

Comment:
I'm wondering if we want to expose both. That is with the config enabled you'd see both buttons. 

Otherwise, how could the user get rid of the card? I guess this is why I'm hung up on language, I'd guessed we wanted both to be shown at the same time.

Reply:
When this feature is enabled, the flow is: Start session > Stop > Delete

I added a sample flow demo in the issue to show how it would work.

![terminate](https://user-images.githubusercontent.com/17336210/203990746-fb99e58c-cc62-40de-a7b5-030d659fba8a.gif)

Reply:
That makes more sense. Thanks.